### PR TITLE
Upgrade to k8scc 6.0.3 and use EnableAWSCNI flag

### PR DESCRIPTION
**Pull request by tfussell**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden [hunk 1]
```diff
@@ -64,7 +64,7 @@
...
...MgQ2FsaWNvIHN0b3JlcyBzb21lIGNvbmZpZ3VyYXRpb24gaW5mb3JtYXRpb24gb24gdGhlIG5vZGUuCiAgLSBhcGlHcm91cHM6IFsiIl0KICAgIHJlc291cmNlczoKICAgICAgLSBub2RlcwogICAgdmVyYnM6CiAgICAgIC0gZ2V0CiAgICAgIC0gbGlzdAogICAgICAtIHdhdGNoCiAgIyBUaGVzZSBwZXJtaXNzaW9ucyBhcmUgb25seSByZXF1cmllZCBmb3IgdXBncmFkZSBmcm9tIHYyLjYsIGFuZCBjYW4KICAjIGJlIHJlbW92ZWQgYWZ0ZXIgdXBncmFkZSBvciBvbiBmcmVzaCBpbnN0YWxsYXRpb25zLgogIC0gYXBpR3JvdXBzOiBbImNyZC5wcm9qZWN0Y2FsaWNvLm9yZyJdCiAgICByZXNvdXJjZXM6CiAgICAgIC0gYmdwY29uZmlndXJhdGlvbnMKICAgICAgLSBiZ3BwZWVycwogICAgdmVyYnM6CiAgICAgIC0gY3JlYXRlCiAgICAgIC0gdXBkYXRlCiAgLSBhcGlHcm91cHM6IFsiY3JkLnByb2plY3RjYWxpY28ub3JnIl0KICAgIHJlc291cmNlczoKICAgICAgLSBibG9ja2FmZmluaXRpZXMKICAgICAgLSBpcGFtYmxvY2tzCiAgICAgIC0gaXBhbWhhbmRsZXMKICAgIHZlcmJzOgogICAgICAtIGdldAogICAgICAtIGxpc3QKICAgICAgLSBjcmVhdGUKICAgICAgLSB1cGRhdGUKICAgICAgLSBkZWxldGUKLS0tCmFwaVZlcnNpb246IHJiYWMuYXV0aG9yaXphdGlvbi5rOHMuaW8vdjEKa2luZDogQ2x1c3RlclJvbGVCaW5kaW5nCm1ldGFkYXRhOgogIG5hbWU6IGNhbGljby1ub2RlCnJvbGVSZWY6CiAgYXBpR3JvdXA6IHJiYWMuYXV0aG9yaXphdGlvbi5rOHMuaW8KICBraW5kOiBDbHVzdGVyUm9sZQogIG5hbWU6IGNhbGljby1ub2RlCnN1YmplY3RzOgogIC0ga2luZDogU2VydmljZUFjY291bnQKICAgIG5hbWU6IGNhbGljby1ub2RlCiAgICBuYW1lc3BhY2U6IGt1YmUtc3lzdGVtCg==",
```
Comment (tfussell) [inline service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden:67]:
```diff
69a70
>     app.kubernetes.io/name: calico-typha
78c79
<     k8s-app: calico-typha
---
>     app.kubernetes.io/name: calico-typha
89a91
>     app.kubernetes.io/name: calico-typha
102a105
>         app.kubernetes.io/name: calico-typha
388a392,393
>     app.kubernetes.io/name: calico-node
>     giantswarm.io/service-type: managed
393c398
<       k8s-app: calico-node
---
>       app.kubernetes.io/name: calico-node
400a406,407
>         app.kubernetes.io/name: calico-node
>         giantswarm.io/service-type: managed
```

File: service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden [hunk 2]
```diff
@@ -100,7 +100,7 @@
...
...9zc2wvY2VydHMKICAgICAgICAgICAgbmFtZTogc3NsLWNlcnRzLWhvc3QKICAgICAgICAgICAgcmVhZE9ubHk6IHRydWUKICAgICAgICAgIC0gbW91bnRQYXRoOiAvZXRjL2t1YmVybmV0ZXMvY29uZmlnLwogICAgICAgICAgICBuYW1lOiBrOHMtY29uZmlnCiAgICAgICAgICAtIG1vdW50UGF0aDogL2V0Yy9rdWJlcm5ldGVzL2t1YmVjb25maWcvCiAgICAgICAgICAgIG5hbWU6IGs4cy1rdWJlY29uZmlnCiAgICAgICAgICAgIHJlYWRPbmx5OiB0cnVlCiAgICAgICAgICAtIG1vdW50UGF0aDogL2V0Yy9rdWJlcm5ldGVzL3NzbAogICAgICAgICAgICBuYW1lOiBzc2wtY2VydHMta3ViZXJuZXRlcwogICAgICAgICAgICByZWFkT25seTogdHJ1ZQogICAgICAgICAgLSBtb3VudFBhdGg6IC9saWIvbW9kdWxlcwogICAgICAgICAgICBuYW1lOiBsaWItbW9kdWxlcwogICAgICAgICAgICByZWFkT25seTogdHJ1ZQogICAgICB2b2x1bWVzOgogICAgICAtIGhvc3RQYXRoOgogICAgICAgICAgcGF0aDogL2V0Yy9rdWJlcm5ldGVzL2NvbmZpZy8KICAgICAgICBuYW1lOiBrOHMtY29uZmlnCiAgICAgIC0gaG9zdFBhdGg6CiAgICAgICAgICBwYXRoOiAvZXRjL2t1YmVybmV0ZXMvY29uZmlnLwogICAgICAgIG5hbWU6IGs4cy1rdWJlY29uZmlnCiAgICAgIC0gaG9zdFBhdGg6CiAgICAgICAgICBwYXRoOiAvZXRjL2t1YmVybmV0ZXMvc3NsCiAgICAgICAgbmFtZTogc3NsLWNlcnRzLWt1YmVybmV0ZXMKICAgICAgLSBob3N0UGF0aDoKICAgICAgICAgIHBhdGg6IC91c3Ivc2hhcmUvY2EtY2VydGlmaWNhdGVzCiAgICAgICAgbmFtZTogc3NsLWNlcnRzLWhvc3QKICAgICAgLSBob3N0UGF0aDoKICAgICAgICAgIHBhdGg6IC9saWIvbW9kdWxlcwogICAgICAgIG5hbWU6IGxpYi1tb2R1bGVzCg==",
```
Comment (tfussell) [inline service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden:103]:
```diff
7c7,8
<     component: kube-proxy
---
>     app.kubernetes.io/name: kube-proxy
>     giantswarm.io/service-type: system
13c14
<       k8s-app: kube-proxy
---
>       app.kubernetes.io/name: kube-proxy
21c22,23
<         component: kube-proxy
---
>         app.kubernetes.io/name: kube-proxy
>         giantswarm.io/service-type: system
```

File: service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden [hunk 3]
```diff
@@ -181,7 +181,7 @@
...
...FzcyB0byBhcHBseSIKZmkKCiMgYXBwbHkgcHJpb3JpdHkgY2xhc3NlczoKUFJJT1JJVFlfQ0xBU1NFU19GSUxFPSJwcmlvcml0eV9jbGFzc2VzLnlhbWwiCgp3aGlsZQogICAgJEtVQkVDVEwgYXBwbHkgLWYgL3Nydi8kUFJJT1JJVFlfQ0xBU1NFU19GSUxFCiAgICBbICIkPyIgLW5lICIwIiBdCmRvCiAgICBlY2hvICJmYWlsZWQgdG8gYXBwbHkgL3Nydi8kUFJJT1JJVFlfQ0xBU1NFU19GSUxFLCByZXRyeWluZyBpbiA1IHNlYyIKICAgIHNsZWVwIDVzCmRvbmUKCiMgYXBwbHkgbmV0d29yayBwb2xpY2llczoKTkVUV09SS19QT0xJQ0lFU19GSUxFPSJuZXR3b3JrX3BvbGljaWVzLnlhbWwiCk5BTUVTUEFDRVM9Imt1YmUtc3lzdGVtIGdpYW50c3dhcm0iCmZvciBuYW1lc3BhY2UgaW4gJHtOQU1FU1BBQ0VTfTsgZG8KICAgIHdoaWxlCiAgICAgICRLVUJFQ1RMIGFwcGx5IC1mIC9zcnYvJE5FVFdPUktfUE9MSUNJRVNfRklMRSAtbiAkbmFtZXNwYWNlCiAgICAgIFsgIiQ/IiAtbmUgIjAiIF0KICAgIGRvCiAgICAgIGVjaG8gImZhaWxlZCB0byBhcHBseSAvc3J2LyRORVRXT1JLX1BPTElDSUVTX0ZJTEUsIHJldHJ5aW5nIGluIDUgc2VjIgogICAgICBzbGVlcCA1cwogICAgZG9uZQpkb25lCgojIGFwcGx5IGs4cyBhZGRvbnMKTUFOSUZFU1RTPSIiCmZvciBtYW5pZmVzdCBpbiAkTUFOSUZFU1RTCmRvCiAgICB3aGlsZQogICAgICAgICRLVUJFQ1RMIGFwcGx5IC1mIC9zcnYvJG1hbmlmZXN0CiAgICAgICAgWyAiJD8iIC1uZSAiMCIgXQogICAgZG8KICAgICAgICBlY2hvICJmYWlsZWQgdG8gYXBwbHkgL3Nydi8kbWFuaWZlc3QsIHJldHJ5aW5nIGluIDUgc2VjIgogICAgICAgIHNsZWVwIDVzCiAgICBkb25lCmRvbmUKZWNobyAiQWRkb25zIHN1Y2Nlc3NmdWxseSBpbnN0YWxsZWQiCg==",
```
Comment (tfussell) [inline service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden:184]:
```diff
87c87
< $KUBECTL delete pods -l k8s-app=kube-proxy -n kube-system
---
> $KUBECTL delete pods -l app.kubernetes.io/name=kube-proxy -n kube-system
103c103
< $KUBECTL -n kube-system -l k8s-app=aws-node wait --for=condition=Ready --timeout=10m pods
---
> $KUBECTL -n kube-system -l app.kubernetes.io/name=aws-node wait --for=condition=Ready --timeout=10m pods
105c105
< $KUBECTL -n kube-system -l k8s-app=calico-node wait --for=condition=Ready --timeout=10m pods
---
> $KUBECTL -n kube-system -l app.kubernetes.io/name=calico-node wait --for=condition=Ready --timeout=10m pods
```

File: service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden [hunk 4]
```diff
@@ -190,7 +190,7 @@
       },
       {
         "contents": {
-          "source": "data:text/plain;charset=utf-8;base64,IyEvYmluL2Jhc2gKRW52RmlsZT0iL2V0Yy9rdWJlbGV0LWVudmlyb25tZW50IgoKIyBzZXQgbWF4IHBvZHMgdmFyaWFibGUKCiMgVGhlIG1heGltdW0gbnVtYmVyIG9mIHBvZHMgb24gQVdTIHdpdGggYXdzLWNuaSBpcyBkZWZpbmVkIGJ5IG1heGltdW0gRU5JIGZvciBpbnN0YW5jZSBhbmQgbWF4aW11bSBJUHMgcGVyIEVOSS4gQ2hlY2sgdGhpcyBsaW5rCiMgVGhpcyBpcyBqdXN0IHNpbXBsaWZ5aW5nIHRoZSB3aG9sZSBmb3JtdWxhLiBTbWFsbCBpbnN0YW5jZXMgY2FuIGhhdmUgbGVzcyBFTklzIGFuZCBsZXNzIElQcyBwZXIgRU5JLgojIGh0dHBzOi8vZ2l0aHViLmNvbS9hd3MvYW1hem9uLXZwYy1jbmktazhzI2VuaS1hbGxvY2F0aW9uCiMgaHR0cHM6Ly9kb2NzLmF3cy5hbWF6b24uY29tL0FXU0VDMi9sYXRlc3QvVXNlckd1aWRlL3VzaW5nLWVuaS5odG1sCgppbnN0YW5jZV9zaXplPSQoY3VybCBodHRwOi8vMTY5LjI1NC4xNjkuMjU0L2xhdGVzdC9tZXRhLWRhdGEvaW5zdGFuY2UtdHlwZSAyPi9kZXYvbnVsbHwgY3V0IC1kXC4gLWYyKQoKaWYgW1sgIiR7aW5zdGFuY2Vfc2l6ZX0iID1+IF4oNHhsYXJnZXw4eGxhcmdlfDl4bGFyZ2V8MTJ4bGFyZ2V8MTZ4bGFyZ2V8MTh4bGFyZ2V8MjR4bGFyZ2UpJCBdXTsgdGhlbgoJTUFYX1BPRFM9MTEwCmVsc2UgCglNQVhfUE9EUz00MApmaQoKCmVjaG8gIk1BWF9QT0RTPSR7TUFYX1BPRFN9IiA+PiAke0VudkZpbGV9Cg==",
```
Comment (tfussell) [inline service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden:193]:
```diff
5d4
< 
18,19d16
< 
< 
```

File: service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden [hunk 5]
```diff
@@ -289,7 +289,7 @@
...
...RQYXRoOiAvZXRjL2t1YmVybmV0ZXMvZW5jcnlwdGlvbi8KICAgICAgbmFtZTogazhzLWVuY3J5cHRpb24KICAgICAgcmVhZE9ubHk6IHRydWUKICAgIC0gbW91bnRQYXRoOiAvZXRjL2t1YmVybmV0ZXMvbWFuaWZlc3RzCiAgICAgIG5hbWU6IGs4cy1tYW5pZmVzdHMKICAgICAgcmVhZE9ubHk6IHRydWUKICAgIC0gbW91bnRQYXRoOiAvZXRjL2t1YmVybmV0ZXMvcG9saWNpZXMKICAgICAgbmFtZTogazhzLXBvbGljaWVzCiAgICAgIHJlYWRPbmx5OiB0cnVlCiAgICAtIG1vdW50UGF0aDogL2V0Yy9rdWJlcm5ldGVzL3NlY3JldHMvCiAgICAgIG5hbWU6IGs4cy1zZWNyZXRzCiAgICAgIHJlYWRPbmx5OiB0cnVlCiAgICAtIG1vdW50UGF0aDogL2V0Yy9rdWJlcm5ldGVzL3NzbC8KICAgICAgbmFtZTogc3NsLWNlcnRzLWt1YmVybmV0ZXMKICAgICAgcmVhZE9ubHk6IHRydWUKICB2b2x1bWVzOgogIC0gaG9zdFBhdGg6CiAgICAgIHBhdGg6IC92YXIvbG9nL2FwaXNlcnZlci8KICAgIG5hbWU6IGFwaXNlcnZlci1sb2cKICAtIGhvc3RQYXRoOgogICAgICBwYXRoOiAvZXRjL2t1YmVybmV0ZXMvZW5jcnlwdGlvbi8KICAgIG5hbWU6IGs4cy1lbmNyeXB0aW9uCiAgLSBob3N0UGF0aDoKICAgICAgcGF0aDogL2V0Yy9rdWJlcm5ldGVzL21hbmlmZXN0cwogICAgbmFtZTogazhzLW1hbmlmZXN0cwogIC0gaG9zdFBhdGg6CiAgICAgIHBhdGg6IC9ldGMva3ViZXJuZXRlcy9wb2xpY2llcwogICAgbmFtZTogazhzLXBvbGljaWVzCiAgLSBob3N0UGF0aDoKICAgICAgcGF0aDogL2V0Yy9rdWJlcm5ldGVzL3NlY3JldHMKICAgIG5hbWU6IGs4cy1zZWNyZXRzCiAgLSBob3N0UGF0aDoKICAgICAgcGF0aDogL2V0Yy9rdWJlcm5ldGVzL3NzbAogICAgbmFtZTogc3NsLWNlcnRzLWt1YmVybmV0ZXMK",
```
Comment (tfussell) [inline service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden:292]:
```diff
6a7,8
>     app.kubernetes.io/name: api-server
>     giantswarm.io/service-type: system
```

File: service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden [hunk 6]
```diff
@@ -307,7 +307,7 @@
...
...R0YWNoVm9sdW1lTGltaXQ9ZmFsc2UKICAgIC0gLS1jb25maWc9L2V0Yy9rdWJlcm5ldGVzL2NvbmZpZy9zY2hlZHVsZXIueWFtbAogICAgLSAtLXY9MgogICAgZW52OgogICAgIC0gbmFtZTogS1VCRV9NQVhfUERfVk9MUwogICAgICAgdmFsdWU6ICIyMCIKICAgIHJlc291cmNlczoKICAgICAgcmVxdWVzdHM6CiAgICAgICAgY3B1OiAxMDBtCiAgICAgICAgbWVtb3J5OiAxMDBNaQogICAgbGl2ZW5lc3NQcm9iZToKICAgICAgaHR0cEdldDoKICAgICAgICBob3N0OiAxMjcuMC4wLjEKICAgICAgICBwYXRoOiAvaGVhbHRoegogICAgICAgIHBvcnQ6IDEwMjUxCiAgICAgIGluaXRpYWxEZWxheVNlY29uZHM6IDE1CiAgICAgIHRpbWVvdXRTZWNvbmRzOiAxNQogICAgdm9sdW1lTW91bnRzOgogICAgLSBtb3VudFBhdGg6IC9ldGMva3ViZXJuZXRlcy9jb25maWcvCiAgICAgIG5hbWU6IGs4cy1jb25maWcKICAgICAgcmVhZE9ubHk6IHRydWUKICAgIC0gbW91bnRQYXRoOiAvZXRjL2t1YmVybmV0ZXMva3ViZWNvbmZpZy8KICAgICAgbmFtZTogazhzLWt1YmVjb25maWcKICAgICAgcmVhZE9ubHk6IHRydWUKICAgIC0gbW91bnRQYXRoOiAvZXRjL2t1YmVybmV0ZXMvc3NsLwogICAgICBuYW1lOiBzc2wtY2VydHMta3ViZXJuZXRlcwogICAgICByZWFkT25seTogdHJ1ZQogIHZvbHVtZXM6CiAgLSBob3N0UGF0aDoKICAgICAgcGF0aDogL2V0Yy9rdWJlcm5ldGVzL2NvbmZpZwogICAgbmFtZTogazhzLWNvbmZpZwogIC0gaG9zdFBhdGg6CiAgICAgIHBhdGg6IC9ldGMva3ViZXJuZXRlcy9rdWJlY29uZmlnCiAgICBuYW1lOiBrOHMta3ViZWNvbmZpZwogIC0gaG9zdFBhdGg6CiAgICAgIHBhdGg6IC9ldGMva3ViZXJuZXRlcy9zc2wKICAgIG5hbWU6IHNzbC1jZXJ0cy1rdWJlcm5ldGVzCg==",
```
Comment (tfussell) [inline service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden:310]:
```diff
6a7,8
>     app.kubernetes.io/name: scheduler
>     giantswarm.io/service-type: system
21a24
>     - --profiling=false
```

File: service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden [hunk 7]
```diff
@@ -298,7 +298,7 @@
...
...M6CiAgICAgIHJlcXVlc3RzOgogICAgICAgIGNwdTogMjAwbQogICAgICAgIG1lbW9yeTogMjAwTWkKICAgIGxpdmVuZXNzUHJvYmU6CiAgICAgIGh0dHBHZXQ6CiAgICAgICAgaG9zdDogMTI3LjAuMC4xCiAgICAgICAgcGF0aDogL2hlYWx0aHoKICAgICAgICBwb3J0OiAxMDI1MQogICAgICBpbml0aWFsRGVsYXlTZWNvbmRzOiAxNQogICAgICB0aW1lb3V0U2Vjb25kczogMTUKICAgIHZvbHVtZU1vdW50czoKICAgIC0gbW91bnRQYXRoOiAvZXRjL2t1YmVybmV0ZXMvY29uZmlnLwogICAgICBuYW1lOiBrOHMtY29uZmlnCiAgICAgIHJlYWRPbmx5OiB0cnVlCiAgICAtIG1vdW50UGF0aDogL2V0Yy9rdWJlcm5ldGVzL2t1YmVjb25maWcvCiAgICAgIG5hbWU6IGs4cy1rdWJlY29uZmlnCiAgICAgIHJlYWRPbmx5OiB0cnVlCiAgICAtIG1vdW50UGF0aDogL2V0Yy9rdWJlcm5ldGVzL3NlY3JldHMvCiAgICAgIG5hbWU6IGs4cy1zZWNyZXRzCiAgICAgIHJlYWRPbmx5OiB0cnVlCiAgICAtIG1vdW50UGF0aDogL2V0Yy9rdWJlcm5ldGVzL3NzbC8KICAgICAgbmFtZTogc3NsLWNlcnRzLWt1YmVybmV0ZXMKICAgICAgcmVhZE9ubHk6IHRydWUKICB2b2x1bWVzOgogIC0gaG9zdFBhdGg6CiAgICAgIHBhdGg6IC9ldGMva3ViZXJuZXRlcy9jb25maWcKICAgIG5hbWU6IGs4cy1jb25maWcKICAtIGhvc3RQYXRoOgogICAgICBwYXRoOiAvZXRjL2t1YmVybmV0ZXMva3ViZWNvbmZpZwogICAgbmFtZTogazhzLWt1YmVjb25maWcKICAtIGhvc3RQYXRoOgogICAgICBwYXRoOiAvZXRjL2t1YmVybmV0ZXMvc2VjcmV0cwogICAgbmFtZTogazhzLXNlY3JldHMKICAtIGhvc3RQYXRoOgogICAgICBwYXRoOiAvZXRjL2t1YmVybmV0ZXMvc3NsCiAgICBuYW1lOiBzc2wtY2VydHMta3ViZXJuZXRlcwo=",
```
Comment (tfussell) [inline service/controller/internal/cloudconfig/testdata/case-0-tccp-test.golden:301]:
```diff
6a7,8
>     app.kubernetes.io/name: controller-manager
>     giantswarm.io/service-type: system
26a29
>     - --profiling=false
```

**Comment by omegas27** [comment]
Nothing to do with your PR but I think we should be careful when releasing those new labels. Not because it is a breaking change but we should remove the k8s-label at some point so should we prepare something about this saying we will remove them in a future release?

File: CHANGELOG.md [hunk 8]
```diff
@@ -16,13 +16,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add mixed instance support for worker ASGs.
+- Add common labels to our managed components.
```
Comment (xh3b4sd) [inline CHANGELOG.md:19]: This needs to go to the unreleased section now since we released the operator.

File: CHANGELOG.md [hunk 9]
```diff
@@ -21,8 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Improve cleanup of `DrainerConfig` CRs after node draining.
 - Use release.Revision in Helm chart for Helm 3 support.
-
-
+- Disable profiling for Controller Manager and Scheduler.
```
Comment (xh3b4sd) [inline CHANGELOG.md:26]: This here does not look right.

Comment (tfussell) [reply CHANGELOG.md:26]: It looks like this was actually changed in [v6.0.3](https://github.com/giantswarm/k8scloudconfig/blob/master/CHANGELOG.md#v603-2020-04-16) so I will move it to unreleased. Thanks.

**Pull request merged**